webpack.config.js

```diff
@@ -62,11 +62,14 @@ module.exports = ({develop}) => ({
     ]
   },
   resolve: {
-    extensions: ['.tsx', '.ts', '.js']
+    alias: {
+      "@modules": path.resolve(__dirname, 'src/modules')
+    },
```

Comment:
Checkout https://www.npmjs.com/package/tsconfig-paths-webpack-plugin for future, it allows you to specify aliases only in one place (`tsconfig.json`) without necessity to duplicate the code in the `webpack.config.js`.